elixir/apps/web/lib/web/live/resources/new.ex

```diff
@@ -103,6 +103,13 @@ defmodule Web.Resources.New do
               disabled={is_nil(@form[:type].value)}
               required
             />
+            <p :if={@form[:type].value == :dns} class="mt-2 text-xs text-neutral-500">
+              To recursively match all subdomains, use a wildcard (e.g. *.company.com).
+              To non-recursively match all subdomains, use a question mark (e.g. ?.company.com).
```

Comment:
I'm wondering if there's a slightly different way to phrase this.  I worry this may still be confusing for some people.  

What do you think about saying something like:
```
DNS resources may use 2 types of wildcards '*' or '?'
The '*' wildcard will match anything to the left of the domain entered (e.g. *.company.com will match foo.company.com AND bar.foo.company.com)
The '?' wildcard will only match one level of subdomain left of the domain entered (e.g. ?.company.com will match foo.company.com but NOT bar.foo.company.com)
```

Doesn't necessarily need to be exactly what I typed, just wondering if it might help the user understand a little easier what we're trying to say.

Reply:
Let me see what I can come up with